jgrapht-core/src/main/java/org/jgrapht/GraphTests.java

```diff
@@ -599,6 +599,36 @@
         return requireDirectedOrUndirected(graph, GRAPH_MUST_BE_DIRECTED_OR_UNDIRECTED);
     }
 
+    /**
+     * Check if the given graph is triangle free or not
```

Comment:
*Test whether the given graph is triangle free or not. A graph is considered triangle free if....

also, make sure you specify what happens in case of directed graphs. E.g, is (u,v)(v,w), (u,w) a triangle, or is the direction of the edges irrelevant?